umh-core/integration/integration_test.go

```diff
@@ -30,33 +30,22 @@ import (
 
 // ---------- Actual Ginkgo Tests ----------
 
-var _ = Describe("UMH Container Integration", Ordered, Label("integration"), func() {
-
-	AfterEach(func() {
-		if CurrentSpecReport().Failed() {
-			fmt.Println("Test failed, printing container logs:")
-			printContainerLogs()
-
-			// Print the latest YAML config
-			fmt.Println("\nLatest YAML config at time of failure:")
-			configPath := getConfigFilePath()
-			config, err := os.ReadFile(configPath)
-			if err != nil {
-				fmt.Printf("Failed to read config file %s: %v\n", configPath, err)
-			} else {
-				fmt.Println(string(config))
-			}
+// Note: Redpanda allocates 2GB of memory per core.
+// Additionally 1.5GB (or 7% of the total memory, whichever is greater) are required to be available, after this allocation to make seastar happy.
+const DEFAULT_MEMORY = "4096m"
```

Comment:
maybe a reference to the same statements in the makefile? so that you know that if you change it here, you also ened to change it somewhere else (and vice versa)? ideally, we only have it in one location, but i'll let you decide